src/test/task.test.js

```diff
@@ -84,3 +85,106 @@ describe('Delete a task method', () => {
     expect(list).toHaveLength(2);
   });
 });
+
+describe('Update the status of a task', () => {
+  test('Update the status of a task with id', () => {
+    // Arrange
+    output = [
+      { description: 'New Task', index: 1, completed: false },
+      { description: 'Third Task', index: 2, completed: true },
+    ];
+
+    // Act
+    updateStatus(tasks, 1, false);
+    updateStatus(tasks, 2, true);
+    // Assert
+    expect(tasks).toMatchObject(output);
+  });
+
+  test('Check if the status of the tasks has been updated in Local Storage', () => {
+    // Act
+    updateStorage(tasks);
+    // Assert
+    expect(localStorage.getItem('storage')).toMatchObject(tasks);
+  });
+});
+
+describe('Remove all completed tasks', () => {
+  test('Remove a completed task in the local storage', () => {
+    // Arrange
+    output = [
+      { description: 'New Task', index: 1, completed: false },
+    ];
+    // Act
+    tasks = removeCompletedTasks(tasks);
+    // Assert
+    expect(tasks).toMatchObject(output);
+    expect(localStorage.getItem('storage')).toMatchObject(tasks);
+  });
+});
+
+describe('Edit a tasks', () => {
+  test('Check If edit task is working', () => {
+    // Arrange
+    const description = 'The Task has been Edited';
+    const id = 1;
+    output = [
+      { description, index: 1, completed: false },
+    ];
+    // Act
+    edit(tasks, description, id);
+    // Assert
+    expect(tasks).toMatchObject(output);
+  });
+});
+
+describe('Update the local storage after Drag/Drop', () => {
+  test('test', () => {
+    // Arrange
+    addTask(tasks, 'second task');
+    addTask(tasks, 'third task');
+    updateStorage(tasks);
+
+    output = [
+      { description: 'third task', index: 1, completed: false },
+      { description: 'The Task has been Edited', index: 2, completed: false },
+      { description: 'second task', index: 3, completed: false },
+    ];
+    // console.log('tasks ', tasks);
+
```

Comment:
- [ ] Kindly clean up the commented `console.log`  in your codebase. It does not make your codebase look tidy.
- [ ] Please document your testing in the README by including the testing process in the README.

[OPTIONAL]

- [ ] You have way too many comments in this module. Most of them are unnecessary, they'll simply make the code untidy. It's clear what each line of code does, so adding comments at every other line may not be necessary. I advise that you remove them.

Reply:
Thanks For Your Comments,
About your optional comment, In the [JS Testing Best Practises - AAA](https://github.com/goldbergyoni/javascript-testing-best-practices#-%EF%B8%8F-12-structure-tests-by-the-aaa-pattern) topic, It says that we should separate each part of AAA with comments. You can check the code examples on the topic.